tests/wake-format/basic/stdout

```diff
@@ -853,8 +899,8 @@ def a =
     # comment
 
     def c = 5
-
     def d = 5
+
     # comment
 
     result
```

Comment:
Much better!  I think you've said previously that if the comment had been followed by two blank lines before `result` then both would stay in the output; is that still true?

Reply:
Yes. `#comment` is bound to `result` and it captures all newlines along the way. We could possibly change that. I'm not sure which is better, but right now we are assuming the author had a reason for not putting them together